python/thunder/rdds/fileio/imagesloader.py

```diff
@@ -81,7 +81,7 @@ def fromStack(self, dataPath, dims=None, dtype=None, ext='stack', startIdx=None,
         recursive: boolean, default False
             If true, will recursively descend directories rooted at datapath, loading all files in the tree that
             have an extension matching 'ext'. Recursive loading is currently only implemented for local filesystems
-            (not s3).
+            (not s3 or Google Storage).
```

Comment:
Total nit, but maybe abbreviate `S3` and `GS` here to match the way you abbreviated the two elsewhere.